src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/repository/TaskResourceRepository.java

```diff
@@ -62,6 +63,10 @@ public interface TaskResourceRepository extends CrudRepository<TaskResource, Str
 
     List<TaskResource> findAllByTaskIdIn(List<String> taskIds, Sort order);
 
+    @Query(value = "select c.task_id AS taskid, c.state AS state from {h-schema}tasks c where c.case_id=:caseId",
```

Comment:
Just to follow the conventions, can this query be moved to a constant ?

Reply:
Done